projects/library/table/src/table.component.spec.ts

```diff
@@ -18,6 +18,9 @@ import {
   getTableInstance,
   TestData,
 } from '@terminus/ui/table/testing';
+import {
+  getFooterCells, getFooterRows,
+} from '../testing/src/test-helpers';
```

Comment:
err... separate lines?

Reply:
Yup, I realized I missed the alignment class so I'll fix this when I push that up.